docs/vulkan.dox

```diff
@@ -205,7 +205,7 @@ an existing Vulkan surface.
 @section vulkan_window Creating the window
 
 Unless you will be using OpenGL or OpenGL ES with the same window as Vulkan,
-there is no need to create a context.  You can disable context creation with the
```

Comment:
This seems to be a stray edit.

Reply:
Yes please. Two spaces are used in all GLFW comments and documentation sources.

Reply:
Done